test/unit/api/no-conflict.js

```diff
@@ -1,9 +0,0 @@
-import skate from '../../../src/index';
```

Comment:
I'm thinking we should still keep the test but just alter it to use the global module.